engines/bastion_katello/app/assets/javascripts/bastion_katello/common/views/katello-agent-notice.html

```diff
@@ -1,7 +1,7 @@
 <section>
   <p bst-alert="warning" >
     <span translate>
-      Katello-agent is deprecated and will be removed in a future release.
+      Katello-agent is deprecated and will be removed in Katello 4.10.
```

Comment:
This hard-coding was missed in the last PR, which updated the removal version in the API deprecation warnings only.